templates/absences/add.html.twig

```diff
@@ -230,7 +227,7 @@
               </td>
             </tr>
 
-            {% if right701 %}
+            {% if right701 == 1 %}
```

Comment:
I don't like this variable name. Right 701 == manage absence documents

Reply:
I re-used a var name set in edit function, I will change the same for more accurency.